models/asymkey/gpg_key_commit_verification.go

```diff
@@ -506,6 +507,9 @@ func CalculateTrustStatus(verification *CommitVerification, repoTrustModel repo_
 		} else {
 			isMember, err = isOwnerMemberCollaborator(verification.SigningUser)
 		}
+		if err != nil {
+			return err
+		}
```

Comment:
It changes the behavior (Just a brief review, I am not sure whether it really affects)

Before: if error occurs, `verification.TrustStatus` still gets values.

After: if error occurs, `verification.TrustStatus` is empty.


I think the old behavior was done by purpose because it is called by ParseCommitsWithSignature

Reply:
well I kinda dislike how it indirect does things ... but I want to keep this pull diff simple ... adoptiong

Reply:
dd540dba4177e82c9ab0a87ac60e6908a181208f